src/runtime-rs/README.md

```diff
@@ -0,0 +1,86 @@
+# runtime-rs
+
+## Architecture overview
+
+`runtime-rs` is a runtime written in Rust, introduced in Kata Containers 3.0, it is composed of several crates.
```

Comment:
Thanks @liubin - this looks great. My only suggestion is to add an explicit "Introduction" section that explains in simple terms:

- What `runtime-rs` is.
- A bit of detail about Kata 3.0
  Maybe including a note that although runtime-rs/Dragonball will be provided in 3.0, it will not be the default for 3.0, _etc_. Alternatively, maybe this doc could point to a roadmap doc or similar to show the evolution of the runtimes, with approximate timescales, list of supported hypervisors, whether the runtime is the default, _etc_.
- How it differs from https://github.com/kata-containers/kata-containers/tree/main/src/runtime
- Advantages / disadvantages, _etc_.